src/compute/src/render/mod.rs

```diff
@@ -646,6 +658,25 @@ where
                 if let Some(token) = &self.shutdown_token {
                     oks = oks.with_token(Weak::clone(token));
                 }
+
+                match max_iter {
+                    Some(max_iter) => {
```

Comment:
Nit: `if let Some(max_iter) = max_iter { ... }` would be slightly more readable.

Reply:
Yes, thanks, changed it.